modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -20,13 +20,36 @@ any changes to expected behavior are noted in the release notes that follow.
 
 
 
+[[v1.1.11]]
+=== Version 1.1.11 (4 October 2023)
+
+This is a regular maintenance release.
+
+https://docs.couchbase.com/sdk-api/couchbase-kotlin-client-1.1.11/index.html[API Reference]
+| http://docs.couchbase.com/sdk-api/couchbase-core-io-2.4.11/[Core API Reference]
+
+The supported and tested dependencies for this release are:
```

Comment:
Do we need this "supported and tested dependencies" section? This info is all in the POM for people who care, and for people who don't care, it's just noise.

Reply:
Yes - I copied that over by mistake   :-/